### ISSUE TITLE
Chromedriver download now FAILS completely

### ISSUE THREAD

**Issue by lprimak** [arquillian/arquillian-extension-drone#381]
Due to changes on the Chromium side, the LICENSE file now tries to get executed instead of the actual chromedriver.
Drone now downloads target/drone/XXX/chromedriver and chromedriver.LICENSE
and chromedriver.LICENSE gets exexuted instead of chromedriver

**Comment by lprimak** [comment]
maybe related to #373

**Comment by jonkoops** [comment]
Would also be fixed by implementing #359 I imagine.

**Comment by lprimak** [comment]
Looks like this is now fixed with Arquillian 14

**Comment by lprimak** [comment]
Tested, however it doesn't work even in Arquillian 14

**Comment by bartoszmajsak** [comment]
@lprimak What is Arquillian 14?

**Comment by lprimak** [comment]
@bartoszmajsak Arquillian 14 -> shorthand for 1.7.0.Alpha14 :)

**Comment by lprimak** [comment]
Just did another test with Arquillian xxx-alpha-14
Looks like it's still trying to execute `LICENSE.chromedriver`
```
lprimak@Lennys-MacBook-Pro 2b6c666b02384e07e0e8b0c122a2f647 % ll
total 35592
-rwxr--r--  1 lprimak  staff    271562 Feb 14 11:38 LICENSE.chromedriver
-rw-r--r--  1 lprimak  staff  17947936 Feb 14 11:38 chromedriver
lprimak@Lennys-MacBook-Pro 2b6c666b02384e07e0e8b0c122a2f647 % 
```

**Comment by lprimak** [comment]
My bad. I was "confusing" arquillian-core with drone.

However, I still can't use updated drone so Chrome works...
Latest alpha that I am using still has issues with Selenium 4 that prevents it from running, as described here:
https://github.com/arquillian/arquillian-extension-drone/issues/272

Is there something I am doing wrong @bartoszmajsak ?

**Comment by stringbean** [comment]
#373 fixed this and was released in 3.0.0-alpha.7. No fix has been released for 2.5.x. I'll backport the changes for this and Apple Silicon support.

**Comment by lprimak** [comment]
@stringbean When I try to run 3.0.0-alpha.7 I get issues described here: https://github.com/arquillian/arquillian-extension-drone/issues/272

Is there anything special you are doing to get this to run?

**Comment by stringbean** [comment]
> @stringbean When I try to run 3.0.0-alpha.7 I get issues described here: #272
> 
> Is there anything special you are doing to get this to run?

I only verified the fix worked with WebDriver and not Graphene. For our CI builds we're using a fork of v2 with the fix backported.

**Comment by melloware** [comment]
MyFaces tests using Arquillian just started getting this last week ` This version of ChromeDriver only supports Chrome version 108`

```
Runtime Unable to instantiate Drone via org.openqa.selenium.chrome.ChromeDriver(ChromeOptions): org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: session not created: This version of ChromeDriver only supports Chrome version 108
Current browser version is 110.0.5481.100 with binary path /usr/bin/google-chrome 
Host info: host: 'fv-az357-369', ip: '10.1.1.53'
Build info: version: '4.7.2', revision: '4d4020c3b7'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.0-1033-azure', java.version: '11.0.18'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [null, newSession {capabilities=[Capabilities {browserName: chrome, browserVersion: 100, chromeDriverVersion: 108.0.5359.71, goog:chromeOptions: {args: [--headless], extensions: []}, platformName: ANY}], desiredCapabilities=Capabilities {browserName: chrome, browserVersion: 100, chromeDriverVersion: 108.0.5359.71, goog:chromeOptions: {args: [--headless], extensions: []}, platformName: ANY}}]
```

**Comment by lprimak** [comment]
@melloware you can work around this by specifying the binary directly (both chrome and chromedriver)
and not using automatic downloads.
It's a little clunky but works better and is more stable than automatic downloads.
Also, firefox work better for headless testing IMHO (firefox -headless)

**Comment by melloware** [comment]
Thank @lprimak I guess we will have to switch to that model.

**Comment by lprimak** [comment]
@bartoszmajsak This is a serious problem. Any chance this gets fixed anytime soon?
Thank you!

**Comment by bartoszmajsak** [comment]
@lprimak I'm a bit confused about the root issue here. Is it not working with the patch brought with #373 or is the problem of not having it backported yet to v2? Or is it a problem occurring in Graphene?

**Comment by lprimak** [comment]
I can't test #373 in v3 because of Graphene issue (https://github.com/arquillian/arquillian-extension-drone/issues/272) and lack of back-port for v2

The end result is that there is no currently working Graphene that functions with chrome at all.

**Comment by bartoszmajsak** [comment]
Thanks @lprimak, I will take care of backporting it.

**Comment by bartoszmajsak** [comment]
@lprimak I've merged fixes from @stringbean into `v2-support` branch. It's not released yet but maybe you can go ahead and try to use jitpack to grab the dependency (or build the snapshot locally).

**Comment by lprimak** [comment]
Looks like the changes work. thank you!
Please release a new version of v2 because it's the only one that works with Graphene currently.

**Comment by lprimak** [comment]
Will be released with Drone 2.5.6

**Comment by bartoszmajsak** [comment]
> Will be released with Drone 2.5.6

@lprimak It's released.
